docs/release-notes.md

```diff
@@ -2,6 +2,23 @@
 
 This page summarizes changes for each major update in Proton and Timeplus Cloud, including new features and important bug fixes. For previous release notes, please check [this page](changelog).
 
+## Apr 1, 2024
+
+*Timeplus Proton:*
+  * Added support for selecting message key from Apache Kafka external streams, via `SELECT _message_key, * FROM external_stream`.
+  * Streaming processing now supports nullable data type.
+  * [External Table](proton-clickhouse-external-table#create-external-table): When naming ClickHouse external tables, special characters such as dashes are now supported. Simply set `table='test-a-b'` in the `CREATE EXTERNAL STREAM` DDL.
```

Comment:
DDL is wrong, should be `CREATE EXTERNAL TABLE`. "When naming ClickHouse tables" is confusing.  The user has already given a name in ClickHouse table, but if the name contains things like dash, previous version of Proton doesn't support it